operations/workplace/README.md

```diff
@@ -28,7 +28,7 @@ The server has two teams:
 
 URL: [https://community.mattermost.com/core](https://community.mattermost.com/core/messages/@github)
 
-Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://pre-release.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator
+Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://pre-release.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator.
```

Comment:
Not strictly part of this PR but the ``pre-release.mattermost.com`` is an old server so not sure we want to link it throughout this README page.
Also, ``https://daily-community.mattermost.com`` should be ``https://community-daily.mattermost.com.

Reply:
Thanks, @amyblais, fixed the community-daily one. 

I'm happy to remove the link to the pre-release server. Do we have a replacement, or can I adjust that sentence to include the other two servers only?

Reply:
The replacement is https://community.mattermost.com.